modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/GeoIpDownloaderIT.java

```diff
@@ -167,6 +168,21 @@ public void testInvalidTimestamp() throws Exception {
             assertFalse(result.getIngestDocument().hasField("ip-asn"));
             assertFalse(result.getIngestDocument().hasField("ip-country"));
         });
+        settingsResponse =
+            client().admin().cluster()
+                .prepareUpdateSettings()
+                .setPersistentSettings(Settings.builder()
+                    .put("ingest.geoip.database_validity", (String) null))
```

Comment:
Minor nit:
```suggestion
                    .putNull("ingest.geoip.database_validity")
```

Reply:
Nice, I haven't seen that method before, I've update it in other places in this test as well